projects/clober-liquidity-vault/index.js

```diff
@@ -8,19 +9,42 @@ const abi = {
 
 const config = {
   base: {
-    rebalancer: '0x13f2Ff6Cc952f4181D6c316426e9CbdA957c6482',
+    rebalancer: '0x6A0b87D6b74F7D5C92722F6a11714DBeDa9F3895',
     bookManager: '0x382CCccbD3b142D7DA063bF68cd0c89634767F76',
```

Comment:
@g1nt0ki 
This is the address of the order book contract. However, if you simply check the total balance of the order book, it will include assets owned by other users. To obtain an accurate TVL, you need to filter out and consider only the orders created by the Rebalancer.

Reply:
but this is still the contract deployed by your team right? wdym by other users?

Reply:
Yes, this is indeed the contract deployed by our team. The BookManager contract provides an order book infrastructure, allowing anyone to place limit orders permissionlessly. When a user places a limit order, the desired amount of input tokens is temporarily locked in the BookManager contract. Once the order is filled, the user can claim and retrieve the output tokens.
The Rebalancer contract is a separate service that operates utilizing the BookManager contract. It acts as a market-making contract on the order book provided by BookManager, using its own assets. Consequently, the total asset value held by Rebalancer is the sum of the assets directly held by Rebalancer and the assets locked by limit orders in the BookManager contract.

Reply:
thanks for the explanation, from our pov (for protocol tvl) we dont distinguish between user depositing for making a trade or as LP, as long as the token is locked in contracts deployed by your team, it is your tvl :)

Reply:
sorry, now I remember the issue here and why adapter is the way it is.

We have already listed "clober v2", so, if we count LP provider in clober v2 from here, it would lead tro doublecounting, do you see the issue?

Reply:
Ah, if there is such a standard regarding double-counting in TVL calculations, the current situation is understandable. In that case, could you please update only the part where the Rebalancer contract address is changed?